lang/migrations/0009_remove_translationcache_lang_transl_text_4a497b_idx_and_more.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.2.19 on 2025-08-11 08:18
+
+import hashlib
+
+from django.db import migrations, models
+
+
+def fill_text_hash(apps, schema_editor):
+    TranslationCache = apps.get_model("lang", "TranslationCache")
+    for obj in TranslationCache.objects.all():
+        if not obj.text_hash:
+            obj.text_hash = hashlib.sha256(obj.text.encode("utf-8")).hexdigest()
+            obj.save(update_fields=["text_hash"])
+    print(" Hash field is populated in lang_translationcache.")
```

Comment:
I recommend doing this in Postgres if possible.

https://www.postgresql.org/docs/current/pgcrypto.html